### ISSUE TITLE
Failed to compile. Also GitHub links not opening

### ISSUE THREAD

**Issue by RaiyanC** [insidesherpa/JPMC-tech-task-2-PY3#42]
I have nvm and npm installed successfully as the commands node -v and npm -v give me the results I am expecting.
But upon npm start I get these errors

<img width="1285" alt="image" src="https://user-images.githubusercontent.com/47450781/78269813-fb509200-7501-11ea-9c2f-e9fc75149cb1.png">

After typing npm start takes me to this with this error.

<img width="838" alt="image" src="https://user-images.githubusercontent.com/47450781/78269992-2dfa8a80-7502-11ea-9a33-7947ca4186ab.png">

Furthermore these two links keep giving me page not found errors

<img width="819" alt="image" src="https://user-images.githubusercontent.com/47450781/78270179-669a6400-7502-11ea-8eb2-c2c10c97f18d.png">

**Comment by izzimo** [comment]
Hey I am getting this error too did you solve it?

**Comment by RaiyanC** [comment]
If you go into JPMC-tech-task-2-PY3 and delete the node_modules directory then replace it with https://drive.google.com/file/d/1amF7C-4mRZCi3Ab5xuH0XCRQubp4aa_t/view . Then run npm start (don't need to do npm install) - that fixed it for me.

**Comment by jbf-insidesherpa** [comment]
@izzimo and @RaiyanC are you both good here?
Also I checked, those links in the slides you mentioned @RaiyanC  are working

**Comment by izzimo** [comment]
Hi unfortunately I continued to run into problems here. I launched the page
and it crashed giving me the error message again, I mapped onto python 2.7
and that didn't fix it either. I wasn't sure what to do after that

**Comment by izzimo** [comment]
![image](https://user-images.githubusercontent.com/63067089/78339591-9c842a80-758c-11ea-8c4f-8a79d01d780b.png)

**Comment by jbf-insidesherpa** [comment]
@izzimo did you try the suggestion @RaiyanC  gave here (which was also in the setup slides) i.e. just copy the node_modules here https://drive.google.com/file/d/1amF7C-4mRZCi3Ab5xuH0XCRQubp4aa_t/view

and replace the node_modules in your repo with the one you downloaded

**Comment by izzimo** [comment]
Yes I did and I still got the error. I think the issue might be with my
path variables I changed them many times as I was originally using python
3.8 but then tried the mapping to 2.7 and I think somewhere alone here I've
messed them up too much

**Comment by izzimo** [comment]
![image](https://user-images.githubusercontent.com/63067089/78357957-4b832f00-75aa-11ea-88f8-4e68dc2c33b5.png)

**Comment by izzimo** [comment]
That is both my path variables at the moment

**Comment by jbf-insidesherpa** [comment]
@izzimo you shouldn't get the error anymore if you just downloaded the the node_modules from the link and went ahead with.**npm start**

your python version shouldn't matter anymore since you're not running npm install anymore

if ever, there should be a different error.

setting your python version Window's path doesn't affect what npm is using btw. for you to affect npm's python you need to do via 
`npm config set python <path_to_python_exe>`
but then if you do the method prescribed for your case, i.e. just copy the node_modules from the external link, you should not need to do **npm install** anymore nor **npm config set python <path_to_python_exe>**. Just npm start will suffice if you follow that.

On the other hand, if you're having problems starting your python server now, it could be because of you messing up your paths. For you to be sure that you're running the correct python version you directly access it by specifying it in the command line via its absolute path e.g.

"C:\Users\Python27\python.exe" <python_file> here.

**Comment by 9916103020** [comment]
@izzimo you should also have visual studio build tools 2007 installed. Please did you have or not?

**Comment by SiddhlodaAyyar** [comment]
Watch this video and if you still get the error comment down in the youtube section and we'll try to solve it.
https://youtu.be/wRAG9i8O_44
